run/GCHP/setCommonRunSettings.sh.template

```diff
@@ -211,6 +211,21 @@ RRTMG_Timestep_sec=10800
 # Mass tuning factor is used in the HEMCO DustDead extenstion for GEOS-Chem
 # benchmarking and is resolution-dependent. We recommend using offline dust
 # instead of the online extension for GCHP science runs.
+#
+# NOTE: We have retuned the dust scaling factors to account for the
```

Comment:
I don't think this note is necessary since it is in the git history.

Reply:
@lizziel: We have simplified the logic where mass tuning factors are set in commit d08e541762be53774d818deaf691a59df4890c47.